software/open-broadcaster-software/en.md

```diff
@@ -13,6 +13,28 @@ To install OBS Studio, either open up the Software Center and type `obs-studio`
 ``` bash
 sudo eopkg install obs-studio
 ```
+## Virtual Camera
+
+OBS includes a virtual webcam device which can be used to send your OBS stream to video conferencing programs. In order to use this on Solus, you must install `v4l2loopback`. There are two versions of `v4l2loopback` available in the repository, and you must choose the version that corresponds to the kernel you have installed.
```

Comment:
Let's change the following:

```
In order to use this on Solus, you must install `v4l2loopback`. There are two versions of `v4l2loopback` available in the repository, and you must choose the version that corresponds to the kernel you have installed.
```

To...

```
In order to use this on Solus, you must first install from our repository a separate kernel module that is responsible for creating the necessary "loopback" device. There are two versions of this software available in the repository, you must choose the version that corresponds to the kernel you have installed.
```

The reason for this is some folks will just immediately install the first package name they see and not read any further, so we should just avoid using it until we provide the respective instructions.